source/app/static/assets/js/iris/case.asset.js

```diff
@@ -299,6 +314,12 @@ function asset_details(asset_id) {
             var data = $('#form_new_asset').serializeObject();
             if (typeof data["ioc_links"] == "string") {
                 data["ioc_links"] = [data["ioc_links"]]
+            } else {
```

Comment:
It goes there if there are multiple ioc linked to the asset at the "update asset" event.